_sass/components/_image-list.scss

```diff
@@ -0,0 +1,25 @@
+.image-list {
+  display: grid;
+  grid-template-columns: repeat(1, min-content 1fr);
+  align-items: center;
+  row-gap: 20px;
+
+  .small {
+    font-size: 85%;
+    margin-right: var(--padding-lg);
+  }
+}
+
+@media (min-width: 35em) {
+  .image-list {
+    row-gap: 40px;
+    grid-template-columns: repeat(2, min-content 1fr);
+  }
+}
+
+@media (min-width: 55em) {
+  .image-list {
+    row-gap: 60px;
+    grid-template-columns: repeat(3, min-content 1fr);
+  }
+}
```

Comment:
**suggestion:** I think these explicit media queries should be unnecessary. The entire layout can be configured to be implicitly adaptive.

* `row-gap` can be an adaptive value such as `var(--block-flow-sm)`.
* `grid-template-columns` can be adjusted implicitly with something like `repeat(auto-fill, minmax(30ch, 1fr))`.